app/views/spotlight/catalog/_edit_default.html.erb

```diff
@@ -5,7 +5,7 @@
       <% current_exhibit.custom_fields.each do |field| %>
         <div class="form-group">
           <%= d.label field.field, field.label %>
-          <%= d.text_field_without_bootstrap field.field, value: document[field.field], class: 'form-control' %>
+          <%= d.text_field_without_bootstrap field.field, value: @document[field.field], class: 'form-control' %>
```

Comment:
:+1: yeah, I noticed inconsistent use in that file already.  I'll push up a change to make them all local vars.